### PR TITLE
Bundle of improvements and corrections

### PR DESCRIPTION
**Pull request by frombetelgeuse**
- Add emojis for warnings/errors
- Change 'dryrun' output
- Handle 'Unauthorized' response
- Add specific language support for 'dryrun'
- Check branch existence before 'dryrun'
- Correct message about missing identity file
- Correct error messages about params

### CODE REVIEW

File: src/main/java/com/crowdin/cli/utils/console/ExecutionStatus.java [hunk 1]
```diff
@@ -5,8 +5,8 @@
 public enum ExecutionStatus {
 
     OK("[OK] ", "\u2714  "),
-    ERROR("[ERROR] ", "\u26D4  "),
-    WARNING("[WARNING] ", "\u26A0  "),
+    ERROR("[ERROR] ", "\u274C  "),
+    WARNING("[WARNING] ", "⚠️  "),
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/cli/utils/console/ExecutionStatus.java:9]: `\u26A0` ?

Comment (frombetelgeuse) [reply src/main/java/com/crowdin/cli/utils/console/ExecutionStatus.java:9]: Unicode \u26A0 in my console(Linux Mint) is turning into a text warning sign(⚠️), but if specify warning sign itself, then it turns into an emoji(yellow warning sign). I don't know how it works in other consoles, but I decided to left it like this.

File: src/main/resources/messages/messages.properties [hunk 2]
```diff
@@ -107,18 +107,24 @@ error.translation_not_exists='%s' translation file doesn't exist in the specifie
 error.upload_translation_to_storage=Failed to upload the translation file to the storage. Please contact our support team for help
 error.upload_translation=Failed to upload the translation file. Please contact our support team for help
 error.configuration_file_is_invalid=Configuration file is invalid. Check the following parameters in your configuration file:
+error.params_are_invalid=Params are invalid. Check the following parameters:
 error.empty_properties_file=Configuration file is empty. Fill it out accordingly to the following requirements: \
 https://support.crowdin.com/configuration-file/#configuration-file-structure
 error.configuration_file_not_exist=Configuration file doesn't exist. Run the 'crowdin init' to generate configuration skeleton
+error.identity_file_not_exist=Specified identity file doesn't exist
```
Comment (andrii-bodnar) [inline src/main/resources/messages/messages.properties]: Identity file '%s' doesn't exist

File: src/main/resources/messages/messages.properties [hunk 3]
```diff
@@ -107,18 +107,24 @@ error.translation_not_exists='%s' translation file doesn't exist in the specifie
...
 error.reading_configuration_file=Failed to read the configuration file. Try to run the application with admin permission
 error.clearing_temp=Failed to clear temporary files
 error.while_checking_base_path=Failed to check base path. Try to run the application with admin permission
 
+error.response.organization_not_found=Organization not found
+error.response.project_not_found=Project with id '%s' doesn't exist
+error.response.unauthorized=Please make sure you entered valid Personal Access Token
```
Comment (andrii-bodnar) [inline src/main/resources/messages/messages.properties]: Failed to authorize in Crowdin. Make sure you entered valid Personal Access Token or try to generate a new one

File: src/main/resources/messages/messages.properties [hunk 4]
```diff
@@ -107,18 +107,24 @@ error.translation_not_exists='%s' translation file doesn't exist in the specifie
 error.upload_translation_to_storage=Failed to upload the translation file to the storage. Please contact our support team for help
 error.upload_translation=Failed to upload the translation file. Please contact our support team for help
 error.configuration_file_is_invalid=Configuration file is invalid. Check the following parameters in your configuration file:
+error.params_are_invalid=Params are invalid. Check the following parameters:
```
Comment (andrii-bodnar) [inline src/main/resources/messages/messages.properties]: Some of the command options are invalid. Check the following parameters:

File: src/main/resources/messages/messages.properties [hunk 5]
```diff
@@ -157,6 +163,8 @@ message.translation_uploaded=Translation file '%s' has been uploaded
 message.branch=Branch '%s'
 message.directory=Directory '%s'
 
+message.warning.not_yml=File with name '%s' has different type from YAML
```
Comment (andrii-bodnar) [inline src/main/resources/messages/messages.properties]: File '%s' is not a YAML or YML file

File: src/main/resources/messages/messages.properties [hunk 6]
```diff
@@ -131,10 +137,12 @@ error.config.translation_has_no_language_placeholders='Translation' parameter sh
 error.config.update_option=Configuration file contains unexpected 'update_option' value. The expected values are 'update_as_unapproved' or 'update_without_changes'
 error.config.escape_quotes=Acceptable values for 'escape_quotes' are: 0, 1, 2, 3. Default is 3. Read more about 'escape_quotes': https://support.crowdin.com/configuration-file-v3/#escape-quotes-options-for-properties-file-format
 error.config.escape_special_characters=Acceptable values for 'escape_special_characters' are: 0 and 1. Default is 1. Read more about 'escape_special_characters': https://support.crowdin.com/configuration-file-v3/#escape-quotes-options-for-properties-file-format
+error.config.params_xor_source_translation=Both the 'source' and the 'translation' must be specified in parameters
 
 message.new_version_text=New version of Crowdin CLI is available! %s -> %s
 message.new_version_text.2=Please update for best experience!
 message.uploading_file=File '%s'
+message.translation_file=Translation file '%s'
```
Comment (andrii-bodnar) [inline src/main/resources/messages/messages.properties:145]: Use existing message `Translation file '%s' has been uploaded`

**Review by andrii-bodnar — APPROVED** [review]

**Pull request merged**